README.md

```diff
@@ -30,13 +30,48 @@ Make sure you have followed the [prerequisites](#prerequisites) specified above
 You should have a development Vault cluster up and running using the [guide](./docs/vault-setup.md) specified above.
 
 
-### Install the Secrets Store CSI Driver (Kubernetes Version 1.15.x+)
+### Install the Secrets Store CSI Driver (Kubernetes Version 1.16.0+)
 
 Make sure you have followed the [Installation guide for the Secrets Store CSI Driver](https://github.com/deislabs/secrets-store-csi-driver#usage)
 
-### Create a SecretProviderClass Resource and a Deployment
+To validate the driver is running as expected, run the following commands:
 
-Update [this sample deployment](examples/v1alpha1_secretproviderclass.yaml) to create a `secretproviderclasses` resource to provide Vault-specific parameters for the Secrets Store CSI driver.
+```bash
+kubectl get pods -l app=csi-secrets-store
+```
+
+You should see the driver pods running on each agent node:
+
+```bash
+NAME                                     READY   STATUS    RESTARTS   AGE
+csi-secrets-store-jlls6                  1/1     Running   0          10s
```

Comment:
nit: This output should be the one with 3 containers in the `csi-secrets-store` pod?

```
NAME                                     READY   STATUS    RESTARTS   AGE
csi-secrets-store-k2256                  3/3     Running   0          10s
```

Reply:
Good catch!